app/assets/v2/js/grants/fund.js

```diff
@@ -70,6 +70,8 @@ $(document).ready(function() {
 
           $('#contributor_address').val(accounts[0]);
 
+          let url;
```

Comment:
@captnseagraves Doesn't look like this url is being assigned a value anywhere ! Do we need it?